apps/provisioning_api/lib/Controller/UsersController.php

```diff
@@ -196,7 +197,14 @@ public function getUsersDetails(string $search = '', ?int $limit = null, int $of
 		$usersDetails = [];
 		foreach ($users as $userId) {
 			$userId = (string) $userId;
-			$userData = $this->getUserData($userId);
+			try {
+				$userData = $this->getUserData($userId);
+			} catch (OCSNotFoundException $e) {
+				// We still want to return all other accounts, but this one was removed from the backends
+				// yet they are still in our database. Might be a LDAP remnant.
+				$userData = null;
+				$this->logger->warning('Still enabled account that is removed from backend, but still exists in Nextcloud database found', ['accountId' => $userId]);
```

Comment:
Feels more log like:
```suggestion
				$this->logger->warning('Found one enabled account that is removed from its backend, but still exists in Nextcloud database', ['accountId' => $userId]);
```